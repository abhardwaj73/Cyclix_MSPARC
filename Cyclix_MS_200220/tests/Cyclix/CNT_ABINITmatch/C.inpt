CELL: 20 0.392699081698724 7.98
FD_GRID: 100 25 40 #(h=0.2)
KPOINT_GRID: 1 16 1
FD_ORDER: 12
BC: d c p
#TWIST_ANGLE: 0.078736657984707 # Twist per unit length
EXCHANGE_CORRELATION: LDA_PW
CHEB_DEGREE: 100
TOL_SCF: 1e-6
TOL_POISSON: 1e-8
TOL_PSEUDOCHARGE: 1e-8
SMEARING: 0.001 
ELEC_TEMP_TYPE: fd
NSTATES: 12
PRINT_FORCES: 1
PRINT_ATOMS: 1
CALC_STRESS: 1
