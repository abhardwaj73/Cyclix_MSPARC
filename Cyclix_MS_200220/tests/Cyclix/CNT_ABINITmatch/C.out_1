***************************************************************************
*                          M-SPARC (Sep 14, 2019)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 29-Oct-2019  Start time: 17:37:40                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 7.980000 
FD_GRID: 120 30 48
FD_ORDER: 12
BOUNDARY_CONDITION: 3
KPOINT_GRID: 1 16 4
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 120
NSTATES: 12
EXCHANGE_CORRELATION: LDA_PW
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Cychel/CNT_ABINITmatch/C.out_1
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.166667 (Bohr)
Mesh spacing in y-direction        :  0.013090 (Bohr)
Mesh spacing in z direction        :  0.166250 (Bohr)
Number of symmetry adapted k-points:  34
Output printed to                  :  ../tests/Cychel/CNT_ABINITmatch/C.out_1
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  16
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  8.59 0.74 8.58
Number of atoms of type 1          :  4
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.2405767189E+00        1.917E-01        1588.404
2            -5.3859784487E+00        2.478E-01        384.414
3            -5.4197410202E+00        9.271E-02        380.897
4            -5.4299115318E+00        2.701E-02        374.873
5            -5.4305658381E+00        7.864E-03        377.623
6            -5.4306468440E+00        3.077E-03        371.383
7            -5.4306549909E+00        2.436E-03        368.036
8            -5.4306605042E+00        1.707E-03        372.696
9            -5.4306666708E+00        7.006E-04        375.101
10           -5.4306668498E+00        9.302E-05        392.978
11           -5.4306668647E+00        4.488E-05        369.206
12           -5.4306669090E+00        1.931E-05        415.168
13           -5.4306668883E+00        8.993E-06        449.999
14           -5.4306669204E+00        3.646E-06        450.315
15           -5.4306669155E+00        3.095E-06        444.081
16           -5.4306669316E+00        1.199E-06        451.519
17           -5.4306669248E+00        9.708E-07        436.105
Total number of SCF: 17    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.4306669248E+00 (Ha/atom)
Total free energy                  : -2.1722667699E+01 (Ha)
Band structure energy              : -7.6038814581E+00 (Ha)
Exchange correlation energy        : -7.3368758281E+00 (Ha)
Self and correction energy         : -6.6930542842E+01 (Ha)
Entropy*kb*T                       : -7.3480110263E-05 (Ha)
Fermi level                        : -1.3421598373E-01 (Ha)
Average force                      :  1.4727883897E+00 (Ha/Bohr)
Maximum force                      :  1.4727898454E+00 (Ha/Bohr)
Time for force calculation         :  68.583 (sec)
Stress (GPa):
 -10430.1472244579 Pressure                           :  3.4767157415E+03 (GPa)
Maximum stress                     :  1.0430147224E+04 (GPa)
Time for stress calculation        :  63.759 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  8668.946 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*        Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
