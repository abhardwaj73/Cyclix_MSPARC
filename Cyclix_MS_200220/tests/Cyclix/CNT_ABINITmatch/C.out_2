***************************************************************************
*                          M-SPARC (Sep 14, 2019)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 30-Oct-2019  Start time: 12:04:22                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 7.980000 
FD_GRID: 120 30 48
FD_ORDER: 12
BOUNDARY_CONDITION: 3
KPOINT_GRID: 1 16 8
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 120
NSTATES: 12
EXCHANGE_CORRELATION: LDA_PW
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Cychel/CNT_ABINITmatch/C.out_2
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.166667 (Bohr)
Mesh spacing in y-direction        :  0.013090 (Bohr)
Mesh spacing in z direction        :  0.166250 (Bohr)
Number of symmetry adapted k-points:  68
Output printed to                  :  ../tests/Cychel/CNT_ABINITmatch/C.out_2
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  16
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  8.59 0.74 8.58
Number of atoms of type 1          :  4
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.2402011986E+00        1.991E-01        3144.664
2            -5.3866729337E+00        2.582E-01        692.908
3            -5.4183503822E+00        1.013E-01        698.605
4            -5.4299873680E+00        3.339E-02        708.192
5            -5.4305475596E+00        7.034E-03        698.990
6            -5.4306184314E+00        3.708E-03        713.642
7            -5.4306250769E+00        2.077E-03        701.778
8            -5.4306305346E+00        1.576E-03        687.515
9            -5.4306336071E+00        5.234E-04        705.374
10           -5.4306345929E+00        1.168E-04        702.252
11           -5.4306344283E+00        3.026E-05        675.682
12           -5.4306344083E+00        2.156E-05        692.197
13           -5.4306344394E+00        1.182E-05        682.614
14           -5.4306344291E+00        4.286E-06        691.430
15           -5.4306344272E+00        3.441E-06        707.167
16           -5.4306344260E+00        5.517E-06        713.368
17           -5.4306344458E+00        2.492E-06        703.948
18           -5.4306344584E+00        7.962E-07        715.957
Total number of SCF: 18    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.4306344584E+00 (Ha/atom)
Total free energy                  : -2.1722537834E+01 (Ha)
Band structure energy              : -7.6016990037E+00 (Ha)
Exchange correlation energy        : -7.3373021455E+00 (Ha)
Self and correction energy         : -6.6930542842E+01 (Ha)
Entropy*kb*T                       : -5.3420420399E-05 (Ha)
Fermi level                        : -1.3484377063E-01 (Ha)
Average force                      :  1.4717109443E+00 (Ha/Bohr)
Maximum force                      :  1.4717115739E+00 (Ha/Bohr)
Time for force calculation         :  117.021 (sec)
Stress (GPa):
-166641.2798896863 Pressure                           :  5.5547093297E+04 (GPa)
Maximum stress                     :  1.6664127989E+05 (GPa)
Time for stress calculation        :  101.786 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  16029.481 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*        Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
