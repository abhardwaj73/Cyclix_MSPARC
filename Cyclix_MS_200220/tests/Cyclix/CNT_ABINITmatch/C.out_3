***************************************************************************
*                          M-SPARC (Nov 27, 2019)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 03-Dec-2019  Start time: 17:30:10                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 7.980000 
FD_GRID: 100 25 40
FD_ORDER: 12
BOUNDARY_CONDITION: 3
KPOINT_GRID: 1 16 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 100
NSTATES: 12
EXCHANGE_CORRELATION: LDA_PW
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Cychel/CNT_ABINITmatch/C.out_3
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.200000 (Bohr)
Mesh spacing in y-direction        :  0.015708 (Bohr)
Mesh spacing in z direction        :  0.199500 (Bohr)
Number of symmetry adapted k-points:  9
Output printed to                  :  ../tests/Cychel/CNT_ABINITmatch/C.out_3
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  16
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.80 0.58 6.78
Number of atoms of type 1          :  4
Estimated total memory usage       :  947.80 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.5729853011E+00        1.515E-01        101.460
2            -5.6824108729E+00        1.563E-01        27.554
3            -5.6950261401E+00        1.369E-01        27.857
4            -5.6995567914E+00        5.493E-02        26.469
5            -5.7005052528E+00        1.445E-02        29.293
6            -5.7005971193E+00        1.201E-02        25.613
7            -5.7006243178E+00        3.735E-03        26.742
8            -5.7006257626E+00        1.684E-03        28.656
9            -5.7006225132E+00        1.861E-04        24.562
10           -5.7006222140E+00        6.870E-05        25.369
11           -5.7006223465E+00        3.510E-05        23.693
12           -5.7006223606E+00        2.284E-05        23.598
13           -5.7006224177E+00        7.095E-06        25.212
14           -5.7006223821E+00        3.147E-06        23.778
15           -5.7006224053E+00        1.316E-06        24.683
16           -5.7006224078E+00        5.804E-07        21.955
Total number of SCF: 16    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.7006224078E+00 (Ha/atom)
Total free energy                  : -2.2802489631E+01 (Ha)
Band structure energy              : -7.9598136811E+00 (Ha)
Exchange correlation energy        : -6.9828059901E+00 (Ha)
Self and correction energy         : -6.6928312310E+01 (Ha)
Entropy*kb*T                       : -1.3821286520E-06 (Ha)
Fermi level                        : -1.6310442701E-01 (Ha)
Average force                      :  6.0655669314E-02 (Ha/Bohr)
Maximum force                      :  6.0658827926E-02 (Ha/Bohr)
Time for force calculation         :  9.340 (sec)
Stress (GPa):
  -7947.2004363184 Pressure                           :  2.6490668121E+03 (GPa)
Maximum stress                     :  7.9472004363E+03 (GPa)
Time for stress calculation        :  9.133 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  551.217 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
