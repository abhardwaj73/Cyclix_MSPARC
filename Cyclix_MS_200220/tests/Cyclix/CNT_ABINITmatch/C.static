***************************************************************************
*                          M-SPARC (Aug 24, 2019)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 28-Oct-2019  Start time: 17:23:19                  *
***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of atoms:
      0.5904002695       0.0000000000       0.0000000000
      0.5904002695       0.0000000000       0.3333333333
      0.5904002695       0.5000000000       0.5000000000
      0.5904002695       0.5000000000       0.8333333333
Atomic forces (Ha/Bohr):
      0.0012207107      -0.0000030161      -0.0606436500
      0.0012210977      -0.0000017826       0.0606422307
      0.0011979479       0.0002399437      -0.0606407926
      0.0011971349       0.0002411626       0.0606422119
