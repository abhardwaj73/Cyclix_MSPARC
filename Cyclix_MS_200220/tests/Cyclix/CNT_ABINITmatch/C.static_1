***************************************************************************
*                          M-SPARC (Aug 24, 2019)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 29-Oct-2019  Start time: 17:37:40                  *
***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of atoms:
      0.5904002695       0.0000000000       0.0000000000
      0.5904002695       0.0000000000       0.3333333333
      0.5904002695       0.5000000000       0.5000000000
      0.5904002695       0.5000000000       0.8333333333
Atomic forces (Ha/Bohr):
      0.0484372762      -0.9160800166       1.1521961649
     -0.1430854772       0.9061324454      -1.1521981021
      0.4898776753      -0.7756093649       1.1521966633
     -0.5675960530       0.7206821243      -1.1521947261
