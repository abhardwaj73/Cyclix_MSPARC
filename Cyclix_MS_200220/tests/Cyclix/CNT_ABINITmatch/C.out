***************************************************************************
*                          M-SPARC (Sep 14, 2019)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 28-Oct-2019  Start time: 17:23:19                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 7.980000 
FD_GRID: 100 25 40
FD_ORDER: 12
BOUNDARY_CONDITION: 3
KPOINT_GRID: 1 16 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 100
NSTATES: 12
EXCHANGE_CORRELATION: LDA_PW
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Cychel/CNT_ABINITmatch/C.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.200000 (Bohr)
Mesh spacing in y-direction        :  0.015708 (Bohr)
Mesh spacing in z direction        :  0.199500 (Bohr)
Number of symmetry adapted k-points:  9
Output printed to                  :  ../tests/Cychel/CNT_ABINITmatch/C.out
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  16
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.63 0.57 6.61
Number of atoms of type 1          :  4
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.5667020617E+00        1.590E-01        97.758
2            -5.6789508770E+00        2.318E-01        27.654
3            -5.6938109949E+00        1.126E-01        27.557
4            -5.6982580843E+00        4.510E-02        25.927
5            -5.6997929127E+00        3.721E-02        26.620
6            -5.7004376775E+00        1.838E-02        26.274
7            -5.7006223825E+00        8.384E-03        25.918
8            -5.7006259424E+00        2.902E-03        25.791
9            -5.7006255005E+00        1.090E-03        24.689
10           -5.7006234972E+00        3.500E-04        25.214
11           -5.7006225069E+00        2.964E-04        25.711
12           -5.7006222082E+00        1.348E-04        23.846
13           -5.7006221644E+00        7.402E-05        26.151
14           -5.7006223080E+00        3.945E-05        26.028
15           -5.7006223419E+00        2.964E-05        23.909
16           -5.7006224157E+00        1.726E-05        23.527
17           -5.7006223950E+00        2.130E-06        24.891
18           -5.7006223899E+00        9.383E-07        22.815
Total number of SCF: 18    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.7006223899E+00 (Ha/atom)
Total free energy                  : -2.2802489560E+01 (Ha)
Band structure energy              : -7.9598177626E+00 (Ha)
Exchange correlation energy        : -6.9828057860E+00 (Ha)
Self and correction energy         : -6.6928312310E+01 (Ha)
Entropy*kb*T                       : -1.3822370944E-06 (Ha)
Fermi level                        : -1.6310473127E-01 (Ha)
Average force                      :  6.0654515868E-02 (Ha/Bohr)
Maximum force                      :  6.0655934820E-02 (Ha/Bohr)
Time for force calculation         :  9.303 (sec)
Stress (GPa):
   -496.6920019154 Pressure                           :  1.6556400064E+02 (GPa)
Maximum stress                     :  4.9669200192E+02 (GPa)
Time for stress calculation        :  9.274 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  598.556 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*        Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
